etc/PrimarySourceSetDataModelSample.json

```diff
@@ -138,7 +138,7 @@
   ],
   "inLanguage": [
     {
-      "@id": "http://www.lexvo.org/page/term/eng/English",
+      "@id": "http://lexvo.org/id/term/eng/English",
```

Comment:
In response to point 4. above:

> It's okay to leave these as schema:Text unless we think we will be storing markdown in them. If we do think that, we can do the type coercion seen on TeachingGuide.text.

We should plan on both `PrimarySource:text` and `PrimarySourceSet:text` being markdown, so I say go ahead and use the same type coercion seen on `TeachingGuide.text`.